src/index.js

```diff
@@ -0,0 +1,96 @@
+import './style.css';
+import addToDo from './modules/addFunction.js';
+import deleteTask from './modules/deleteFunction.js';
+import editTask from './modules/editFunction.js';
+import clearAllCheck from './modules/clearButton.js';
+import checkStatus from './modules/checkStatus.js';
+
+let tasks = [];
+
+const list = document.getElementById('list');
+const container = document.createElement('div');
+container.classList.add('container');
+list.appendChild(container);
+const title = document.createElement('h1');
+title.innerHTML = "Today's To Do";
+const ulList = document.createElement('ul');
+container.appendChild(title);
+const addTask = document.createElement('input');
+addTask.type = 'text';
+addTask.classList.add('addTask');
+addTask.placeholder = 'Add to your list ...';
+container.appendChild(addTask);
+container.appendChild(ulList);
+const clearBtn = document.createElement('button');
+clearBtn.innerHTML = 'Clear all completed';
+container.appendChild(clearBtn);
+
+const display = (taskObj) => {
+  const liTask = document.createElement('li');
+  liTask.classList.add('index');
+  liTask.dataset.index = taskObj.index;
+  const check = document.createElement('input');
+  check.type = 'checkbox';
+  check.classList.add('checkbox');
+  const descriptionTask = document.createElement('input');
+  descriptionTask.classList.add('description');
+  descriptionTask.value = `${taskObj.description}`;
+  const divMenu = document.createElement('div');
+  divMenu.classList.add('divMenu');
+  const span1 = document.createElement('circle');
+  span1.classList.add('menu');
+  const span2 = document.createElement('circle');
+  span2.classList.add('menu');
+  const span3 = document.createElement('circle');
+  span3.classList.add('menu');
```

Comment:
**OPTIONAL
- [ ] Please rethink how these `circle` elements should be named so anyone can understand the name of their respective variables.
Also, since hard coding is not scalable, kindly consider using an image or an SVG to replace these dots into a singular element.